File: client/src/models/Vault.js
import { DatabaseDefault } from "./DatabaseDefault.js";

export class Vault extends DatabaseDefault{
  constructor(d){
    super(d)
    this.name = d.name
    this.description = d.description
    this.img = d.img
    this.isPrivate = d.isPrivate
    this.creatorId = d.creatorId
    this.creator = d.creator

  }
}

File: server/Models/Vault.cs
namespace keeper.Models;

public class Vault : DatabaseDefault
{
  public string Name { get; set; }
  public string Description { get; set; }
  public string Img { get; set; }
  public bool? IsPrivate { get; set; }
  public string CreatorId { get; set; }
  public Account Creator { get; set; }
}